sdk/src/Services/DynamoDBv2/Custom/DocumentModel/Table.cs

```diff
@@ -472,7 +486,28 @@ public static Table LoadTable(IAmazonDynamoDB ddbClient, string tableName, bool
         /// <returns>Table object representing the specified table.</returns>
         public static Table LoadTable(IAmazonDynamoDB ddbClient, string tableName, DynamoDBEntryConversion conversion, bool isEmptyStringValueEnabled)
         {
-            var config = new TableConfig(tableName, conversion, DynamoDBConsumer.DocumentModel, storeAsEpoch: null, isEmptyStringValueEnabled: isEmptyStringValueEnabled);
+            var config = new TableConfig(tableName, conversion, DynamoDBConsumer.DocumentModel, storeAsEpoch: null, isEmptyStringValueEnabled: isEmptyStringValueEnabled, metadataCachingMode: MetadataCachingMode.Default);
+
+            return LoadTable(ddbClient, config);
+        }
+
+        /// <summary>
+        /// Creates a Table object with the specified name, using the
+        /// passed-in client to load the table definition.
+        /// 
+        /// This method will throw an exception if the table does not exist.
+        /// </summary>
+        /// <param name="ddbClient">Client to use to access DynamoDB.</param>
+        /// <param name="tableName">Name of the table.</param>
+        /// <param name="conversion">Conversion to use for converting .NET values to DynamoDB values.</param>
+        /// <param name="isEmptyStringValueEnabled">If the property is false, empty string values will be interpreted as null values.</param>
+        /// <param name="metadataCachingMode">The document API relies on an internal cache of the DynamoDB table's metadata to construct and validate 
+        /// requests. This controls how the cache key is derived, which influces when the SDK will call DescribeTable
```

Comment:
nit: typo in "influces"

Reply:
nit: add doc ref to "DescribeTable"

Reply:
Addressed in https://github.com/aws/aws-sdk-net/compare/8a4f5a780d437debec1f38696b9dc2f069b991c1..1161dc30f02a5a18108c28b0191f21da5080a368